src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -686,6 +686,13 @@ export class CodeModelCliImpl implements CodeModelAz {
         return this.MethodParameter.schema.type;
     }
 
+    private isComplexSchema(type: string): boolean {
+        if (type == SchemaType.Array || type == SchemaType.Object || type == SchemaType.Dictionary || type == SchemaType.Any)
```

Comment:
Can you merge master into your branch? I think Rodge has already done some changes about type any before.
I think you can keep this, but add one more condition into it. when a parameter is manually set as json, this should also be treated as ComplexSchema. right?

Reply:
Hi, just merged and add manual json setting check.